docs/.vitepress/theme/components/Home.vue

```diff
@@ -272,23 +278,23 @@ onMounted(() => {
     <div class="kit-card flex flex-col">
       <div class="h4 !font-bold !text-salmon">1</div>
       <div class="mt-8 flex flex-col flex-1 justify-between">
-        <p class="p2">Download and install Kit CLI.</p>
-        <a href="/docs/cli/installation/" v-ga-track="{ category: 'button', label: 'install', location: 'get started' }" class="kit-button kit-button-salmon md:w-fit mt-6">Install the CLI</a>
+        <p class="p2">Integrate Kit into your CI/CD pipeline using our GitHub Actions, GitLab CI components, or Jenkins plugins.</p>
+        <a href="/docs/cli/installation/" v-ga-track="{ category: 'button', label: 'install', location: 'get started' }" class="kit-button kit-button-salmon md:w-fit mt-6">CI/CD DOCUMENTATION</a>
```

Comment:
Is this supposed to be all CAPS? Also the link is to `cli/installation` is that the correct link?